Controller/Callback/Webhook.php

```diff
@@ -24,7 +24,7 @@ class Webhook extends Action
     protected $request;
 
     private $supportedEvents = [
-        'charge.complete',
+        'charge.capture',
```

Comment:
will this event trigger for auto capture? As per [documentation](https://www.omise.co/api-webhooks#charge-events) it seem like will trigger only on manual capture.

Reply:
I am lookin into it. Thanks for pointing this out.